src/main/java/frc/team166/robot/subsystems/Drive.java

```diff
@@ -64,16 +66,17 @@
     });
 
     final static double AUTOMATIC_ROBOT_FORWARD_SPEED = .2;
-    final static double ABSOLUTE_TOLERANCE_ANGLE = 3;
+    final static double ABSOLUTE_TOLERANCE_ANGLE = 1;
 
     //this makes children that control the tempestGyro, drive motors, and PIDController loop. 
     public Drive() {
 
         SmartDashboard.putData("XBox", xboxArcade());
         SmartDashboard.putData("Turn -45", turnByDegrees(-45));
```

Comment:
We can probably remove the turn -45.
We should also change `"Drive 2 Feet"` to something like `"Stay within 2 feet"`.